main.go

```diff
@@ -108,6 +109,30 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 
 	cache.add(m.ID, *m.Message)
 
+	isByModerator := false
+	for _, r := range m.Member.Roles {
+		if r == env.RoleMod {
+			isByModerator = true
+		}
+	}
+
+	if !isByModerator {
+		matchedString, err := db.FindBlockedWordMatch(m.Message.Content)
+		if err != nil {
+			log.Printf("Failed to check if message \"%s\" contains blocked words\n%s\n", m.Content, err)
+			return
```

Comment:
in case of bot or database failure no one will be able to send commands

probably should move the whole body of `if !isByModerator {` to a separate function